CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Prefix your items with `(Template)` if the change is about the template and not
 - Updated Configuration documentation and uno workaround comment.
 - Cleanup (fixes warnings on Windows, fixes vulnerabilities and renamed extensions) & Update packages.
 - Updated the Segoe MDL2 Assets font to fix an issue with the Flip View icons on Windows.
+- Optimized the .NET workloads install process.
```

Comment:
Use `fix:`conventional commit to generate a new version. `ci:` should only be used to change github-only configuration.

Reply:
Mhh I did not know that, I though any CI-related change was fine.

Reply:
Done